lib/core/msdf-sys/src/lib.rs

```diff
@@ -0,0 +1,184 @@
+extern crate js_sys;
+extern crate wasm_bindgen;
+extern crate vector2d;
+
+mod internal;
+
+use internal::{
+    emscripten_call_function,
+    msdfgen_generate_msdf,
+    msdfgen_free_font,
+    emscripten_data_types
+};
+use js_sys::Uint8Array;
+use wasm_bindgen::JsValue;
+use vector2d::Vector2D;
+use crate::internal::copy_f32_data_from_msdfgen_memory;
+
+// ============
+// === Font ===
+// ============
+
+pub struct Font {
+    handle: JsValue
+}
+
+impl Font {
+    // Loading font from memory
+    //
+    // Loads font from a any format which freetype library can handle.
+    // See https://www.freetype.org/freetype2/docs/index.html for reference
+    pub fn load_from_memory(data: &[u8]) -> Self {
+        let param_types = js_sys::Array::of2(
+            &JsValue::from_str(emscripten_data_types::ARRAY),
+            &JsValue::from_str(emscripten_data_types::NUMBER)
+        );
+        unsafe { // Note [Usage of Uint8Array::view]
```

Comment:
Please see https://github.com/luna/enso/blob/master/doc/rust-style-guide.md#source-notes
Especially: `Any unsafe usage of a function must be accompanied by a source note that explains what makes this particular usage safe.`

Reply:
An there is? // Note [Usage of Uint8Array::view]

Reply:
Oh, hmm, we've been pasting the comments in the place of code, we never used the note-style. But I like it. We can add it to our code style.